app/views/organizations/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1 class="page-header">
+  <%= @organization.login %>
+</h1>
+
+<%= link_to 'Remove Classroom', @organization, class: 'btn btn-sm btn-danger', role: 'button',
```

Comment:
Would it be possible to add some sort of hook on to the organization to fire an event so that the app could update who has permissions?